ui/pages/swaps/select-quote-popover/index.scss

```
@@ -99,15 +99,15 @@
 
     &--selected {
       color: var(--white);
-      background: linear-gradient(90deg, var(--Blue-500) 0%, var(--Blue-400) 101.32%);
+      background: linear-gradient(90deg, var(--text-primary-color) 0%, var(--Blue-400) 101.32%);
```

This is going to flip the gradient from blue500-blue400 to blue500-blue600 but I think it's fine. Dark mode it will look better too because it will be blue400-blue300
```suggestion
      background: linear-gradient(90deg, var(--color-primary-default) 0%, var(--primary-alternative) 101.32%);
```